Formula/dafny.rb

```diff
@@ -0,0 +1,41 @@
+class Dafny < Formula
+  desc "Verification-aware programming language"
+  homepage "https://www.microsoft.com/en-us/research/project/dafny-a-language-and-program-verifier-for-functional-correctness"
+  url "https://github.com/dafny-lang/dafny/archive/v2.3.0.tar.gz"
+  sha256 "ea7ae310282c922772a46a9a85e2b4213043283038b74d012047b5294687d168"
+
+  depends_on "mono"
+  depends_on "mono-libgdiplus"
+  depends_on "nuget"
```

Comment:
does it depend on this at runtime as well?

Reply:
Yeah, I just removed this dependency